solutions/arrays/reduce-array.js

```diff
@@ -0,0 +1,35 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+// Use readLine() for taking input, it will read one line of from the input  and is stored in string format
+let n=parseInt(readLine())
+let answer=parseInt(readLine());
+if(arraySize===1)
+{
+	console.log(answer);
+}
+else
+{
+	answer=answer+parseInt(readLine());
+	for(let index=2;index<nindex++)

```

Comment:
@DevratAgarwal 
1. This can be further simplified by starting the index from `1`. That way you can avoid the `if else`.
2. Please format as per this comment - https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions/pull/272#pullrequestreview-915443064